packages/filesystem/src/browser/file-dialog/file-dialog-tree-filters-renderer.tsx

```diff
@@ -36,11 +36,14 @@ export class FileDialogTreeFilters {
 
 export class FileDialogTreeFiltersRenderer extends ReactRenderer {
 
+    readonly filters: FileDialogTreeFilters;
+
     constructor(
-        readonly filters: FileDialogTreeFilters,
```

Comment:
I think we can keep the readonly and update it to:

```ts
protected readonly filters: FileDialogTreeFilters;

constructor(
  readonly _filters: FileDialogTreeFilters,
  readonly fileDialogTree: FileDialogTree
) {
  super();
  this.filters = { 'All Files': [], ..._filters, };
}
```

Reply:
@vince-fugnitto, 👍 I like the idea of keeping both around, since it may aid with future customization. I have pushed a change that distinguishes between `suppliedFilters` and `appliedFilters`.